website/src/css/customTheme.scss

```diff
@@ -399,24 +399,24 @@ html[data-theme="dark"] article .badge {
 
 /* Admonitions & notes */
 
-.alert--warning {
-  background-color: var(--rn-note-background);
-  font-size: 14px;
-  text-align: center;
+.admonition,
+.theme-admonition {
+  font-size: var(--ifm-blockquote-font-size);
+  margin-bottom: 1.5rem !important;
 
-  .margin-top--md {
-    margin-top: 0.33rem !important;
```

Comment:
These styles have been correctly reset for the general case (bug introduced in https://github.com/facebook/react-native-website/pull/2263/commits/7a22db1cb93d66f7033469b3f19e60fd3e581d91), but we still want to preserve this for `.theme-doc-version-banner` — please can we move these rules into the below selector (line 493)? :)

<img width="494" alt="image" src="https://user-images.githubusercontent.com/2547783/183453529-0a723972-d34a-478d-bf15-de63c603b114.png">